### ISSUE TITLE
Laravel 5.5 compatibility

### ISSUE THREAD

**Issue by andreich1980** [Belphemur/laravel-gettext#4]
Thank you very much for the fork for L5.4 version 👍 
Now it's the time for 5.5 :)

Error when run a command `php artisan gettext:update`

```
  [ReflectionException]
  Method Xinax\LaravelGettext\Commands\GettextUpdate::handle() does not exist
```

According to this https://laravel.com/docs/5.5/upgrade `fire` methods have to be renamed to `handle`.

It worked for me. Would you make the changes?

**Comment by Belphemur** [comment]
Hello @andreich1980 

Thanks for the report, I had already a branch ready for this.
I'm going to release it with the fixes.

**Comment by codiiv** [comment]
@andreich1980 You just saved me maybe a few minutes of work. I run into the issue exactly as you and changing  `fire` to `handle` worked also for me. 
@Belphemur Keep the good job up!

**Comment by Belphemur** [comment]
The version 6.0.0 should have all the fixes for laravel 5.5
